src/mcom/manage-stock-aggregates.md

```diff
@@ -0,0 +1,30 @@
+---
+ee_only: true
+title: Manage Stock Aggregates
+---
+
+Manage Stock Aggregates provides information about existing stock aggregates, and allows you to add new stock aggregates or edit or delete existing ones.
+
+![]({% link images/images-ee/mcom-manage-stock-aggregates.png %}){: .zoom}
+_Manage Stock Aggregates_
+
+You [create sources](https://omsdocs.magento.com/en/features-processes/stock-sourcing/inventory/#configure-sources) and [aggregates](https://omsdocs.magento.com/en/features-processes/stock-sourcing/inventory/#configure-stock-aggregates) in the Order Management System (OMS) Admin. Then you assign which website(s) are associated with each aggregate via Manage Stock Aggregates in the Magento Commerce (MC) Admin.
+
+## Add a new stock aggregate
+
+1. On the _Admin_ sidebar, go to **MCOM** > **Manage Stock Aggregates**.
+1. Click **Add New Stock Aggregate**.
+1. Enter a **Stock code** (name) for this stock aggregate.
+
+   {:.bs-callout-info}
+   The stock code you enter should match the [stock aggregate code](https://omsdocs.magento.com/en/features-processes/stock-sourcing/inventory/#configure-stock-aggregates) in the OMS.
+
+1. Choose a **Website** from the populated list.
+1. When complete, click **Save Stock Aggregate**.
```

Comment:
Don't know if this is a button or not since there isn't a screen.

Reply:
Since there isn't a screen? You mean screenshot/image? I try to not put too many screenshots/images in a topic because they are difficult to maintain. Do you feel it needs one?

Reply:
I just meant I did not know if it was a link or a button or what. 

As far as when to put a screen -- I don't like duplicate screens throughout the topic (like with one minor difference), but for the procedure if there is a dialog/UI page that is not reflected elsewhere, I think it is generally a good call to show it so that users have their bearings and can confirm that "yes, I am in the right place".

Reply:
Gotcha. Revising.